core/commands/config.go

```diff
@@ -502,6 +503,12 @@ func editConfig(filename string) error {
 		return errors.New("ENV variable $EDITOR not set")
 	}
 
+	if runtime.GOOS == "windows" {
+		cmd := exec.Command("cmd", "/C", editor, filename)
+		cmd.Stdin, cmd.Stdout, cmd.Stderr = os.Stdin, os.Stdout, os.Stderr
+		return cmd.Run()
+	}
+
 	cmd := exec.Command("sh", "-c", editor+" "+filename)
```

Comment:
much dupped code
```suggestion
	var cmd *exec.Cmd
	switch runtime.GOOS {
	case "windows":
		cmd = exec.Command("cmd", "/C", editor, filename)
	default:
		cmd = exec.Command("sh", "-c", editor+" "+filename)
	}
```
This code would be cleaner if golang had control flow as expressions but it doesn't.